socrates/lib/site/sponsorpairs.js

```diff
@@ -1,19 +1,21 @@
 'use strict';
 
 const sponsors = require('./sponsors.json');
-const _ = require('lodash');
+const R = require('ramda');
+
+function shuffle(array) {
```

Comment:
Find ich immer etwas wider den open source gedanken.
Hier hast Du eine äquivalente Quelle:
http://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array
Würde es Dir was ausmachen, den Link dort als Kommentar zu hinterlegen?